packages/drivers/odsp-driver/src/createFile/createNewContainerOnExistingFile.ts

```diff
@@ -67,6 +68,12 @@ export async function createNewContainerOnExistingFile(
 
 	const initialUrl = `${baseUrl}/opStream/snapshots/snapshot`;
 
+	const additionalHeaders: { [key: string]: string } = {};
```

Comment:
Nit: this `{ [key: string]: string }` type is used in a handful of places. Might be worth creating a type definition somewhere and documenting it.

Reply:
> is [already] used in a handful of places

That's exactly why I kept using it 😆 

I can replace it with `Record<string, string>` if that's preferable, or export a new type like `export type RequestHeaders = { [key: string]: string }` if that would be better. Let me know

Reply:
Ah, okay. Didn't realize it was an existing pattern. Future cleanup possibility.